### PR TITLE
Custom font sizing

### LINKED ISSUE #2: Optional font size
# About

Font-size should be configurable and one should be able to change it easily without changing tens of lines of CSS code.

**Note:** Relative sizes of the fonts must stay the same.

### PR DESCRIPTION
**Pull request by abettermap**
Closes #2

### CODE REVIEW

**Comment by abettermap** [comment]
@smddzcy I believe this is what you were looking for, but let me know if I missed anything. I also included some brief trips in the `readme`. Could maybe integrate your colors instructions into that section as well, if you think that makes sense.

Also, if you approve the merge, beforehand could you please re-apply the *hacktoberfest* label  approving? I am shooting for the [DigitalOcean t-shirt](https://hacktoberfest.digitalocean.com/) and only have one more merge to go. 😄

**Comment by abettermap** [comment]
@smddzcy Actually disregard the _hacktoberfest_ thing- they actually want four pull requests total and the deadline is tomorrow sooo guess I missed out. Happy I could help anyways though. Neat project!

File: Gruntfile.js [hunk 1]
```diff
@@ -70,6 +70,12 @@ module.exports = function(grunt) {
                 return "url('" + imgUrl + "') repeat";
               }
               return 'initial';
+            },
+            assignFontSize: function(fontSize) {
+              if (fontSize) {
+                return fontSize + 'rem' || 'initial';
```
Comment (smddzcy) [inline Gruntfile.js]: I think this should be `return fontSize + 'rem'`.

Comment (smddzcy) [inline Gruntfile.js]: Thank you so much, you've done very well 👍

Here are the missing things. I can finish it up but if you want to continue:
- All `px` units in LESS/CSS files should be replaced with `rem`s.
- `font_sizes.json` should contain the default sizes so that user will have an idea.

Btw, Hacktoberfest needs 4 pull-requests, not 4 merges, so you still have some chance 😄

**Comment by smddzcy** [comment]
Thank you so much, you've done very well 👍

Here are the missing things. I can finish it up but if you want to continue:
- All `px` units in LESS/CSS files should be replaced with `rem`s.
- `font_sizes.json` should contain the default sizes so that user will have an idea.

Btw, Hacktoberfest needs 4 pull-requests, not 4 merges, so you still have some chance 😄

File: Gruntfile.js (hunk 1 above)
Comment (abettermap) [inline Gruntfile.js]: Oops, you're right. I copied that block of text from the colors method and evidently did not edit that part.

Comment (abettermap) [inline Gruntfile.js]: Ok I made the changes in my fork. So how does this work, do I make another pull request or does this one stay open?

**Comment by abettermap** [comment]
@smddzcy Ha, good to know about the t-shirt, but I think a full day at work and Halloween tonight will prevent any more requests for me. Next year! 😃

File: Gruntfile.js (hunk 1 above)
Comment (smddzcy) [inline Gruntfile.js]: This one stays open, you don't need to do anything. Thank you 👍
